src/app/util/af-main-common.cpp

```diff
@@ -647,31 +647,33 @@ EmberStatus emAfSend(EmberOutgoingMessageType type, MessageSendDestination desti
         status = chipSendUnicast(binding.nodeId, apsFrame, messageLength, message);
         break;
     }
-    case EMBER_OUTGOING_VIA_ADDRESS_TABLE:
+    case MessageSendDestination::VariantViaAddressTable::VariantId:
```

Comment:
Again, the public API should probably be an enum in MessageSendDestination that has the different types.  The internal types we use for the variant should ideally all be private.

Reply:
Fixed